tests/example_netlist.py

```diff
@@ -314,6 +314,9 @@ def example_physical_netlist():
         stubs=[],
         net_type=PhysicalNetType.Vcc)
 
+    phys_netlist.add_physical_cell(cell_name='ibuf', cell_type='port')
+    phys_netlist.add_physical_cell(cell_name='obuf', cell_type='port')
```

Comment:
cell_type should be the physical_netlist.PhysicalCellType enum, not a string.

Reply:
Right, fixed